src/hooks/usePlayerMove.ts

```
@@ -38,19 +38,26 @@ interface PlayerMoveResult {
 
 function getFinishResult(textArray: string[]): string {
   // if we have %, we are on the finish tile. Let's get a random result.
-  const finishValues = textArray.filter((n) => n).map((line) => line.split(': '));
+  // Lines look like "Label 42%" (buildGame.ts joins label + percent with a space,
+  // not a colon) so pull the trailing number off each line instead of splitting on ': '.
+  const finishValues = textArray
+    .filter((n) => n)
+    .map((line) => {
+      const match = line.trim().match(/^(.*?)[\s:]+(-?\d+)$/);
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)